src/Psy/ExecutionLoop/Loop.php

```diff
@@ -21,7 +21,7 @@
 class Loop
 {
     /**
-     * The non-forking loop doesn't have much use for Configuration :)
+     * The non-forking loop doesn't have much use for Configuration :).
```

Comment:
then even it might look wrong, csfixer wants it this way. I am not really sure about what to do about this kind of rules, if i rollback to the prior state travis will refuse the builds.